<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saveGifForm</class>
 <widget class="QWidget" name="saveGifForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#saveGifForm{
background-color: rgb(46, 52, 54);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_screenRecord">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox#groupBox_screenRecord
{
color: rgb(243, 243, 243);
border: 2px solid red;
border-radius:5px;
font:bold 28px;
 margin-top:2ex;
font-family:楷体;
font:bold 28px;
}
 
QGroupBox#groupBox_screenRecord::title
{
subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 3px;
}
QGroupBox#groupBox_screenRecord::indicator {
    /* 选择框尺寸 */
    width:23px;
    height:23px;
}
/*这是对CheckBox的样式进行设定*/
/*QGroupBox::indicator:unchecked {
    image: url(:/Ampctrl/Recources/image/UnCheck_.png);
 /*image: url(:/Ampctrl/Recources/image/UCheckIn.png);
}*/
/*QGroupBox::indicator:checked {
    image: url(:/Ampctrl/Recources/image/Check_.png);
 image: url(:/Ampctrl/Recources/image/CheckIn.png);
} */</string>
       </property>
       <property name="title">
        <string>录屏</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_allselect">
            <property name="styleSheet">
             <string notr="true">QCheckBox#checkBox_allselect{
   	color: rgb(243, 243, 243);
font:18px;
}

QCheckBox#checkBox_allselect::indicator {
    /* 选择框尺寸 */
    width:18px;
    height:18px;
	color: rgb(211, 215, 207);
}
/*这是对CheckBox的样式进行设定*/
QCheckBox#checkBox_allselect::indicator:unchecked
{
    image: url(:/resource/icon/unchecke.png);
    width:18px;
    height:18px;
	color: rgb(211, 215, 207);
}
QCheckBox#checkBox_allselect::indicator:checked
{
	image: url(:/resource/icon/checked.png);
    width:18px;
    height:18px;
	color: rgb(211, 215, 207);
}</string>
            </property>
            <property name="text">
             <string>  全选</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_leftScreen">
            <property name="styleSheet">
             <string notr="true">QCheckBox#checkBox_leftScreen{
   /* color:red;
    background-color:rgb(101,101,101);*/
	color: rgb(243, 243, 243);
    font: 18px;
}

QCheckBox#checkBox_leftScreen::indicator {
    /* 选择框尺寸 */
    width:18px;
    height:18px;
}
/*这是对CheckBox的样式进行设定*/
QCheckBox::indicator:unchecked
{
    image: url(:/resource/icon/unchecke.png);
    width:18px;
    height:18px;
}
QCheckBox::indicator:checked
{
	image: url(:/resource/icon/checked.png);
    width:18px;
    height:18px;
}</string>
            </property>
            <property name="text">
             <string>  左屏幕</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_left">
            <property name="styleSheet">
             <string notr="true">color: rgb(243, 243, 243);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_mainScreen">
            <property name="styleSheet">
             <string notr="true">QCheckBox#checkBox_mainScreen{
   	color: rgb(243, 243, 243);
font:18px;
}

QCheckBox#checkBox_mainScreen::indicator {
    /* 选择框尺寸 */
    width:18px;
    height:18px;
	color: rgb(211, 215, 207);
}
/*这是对CheckBox的样式进行设定*/
QCheckBox#checkBox_mainScreen::indicator:unchecked
{
    image: url(:/resource/icon/unchecke.png);
    width:18px;
    height:18px;
	color: rgb(211, 215, 207);
}
QCheckBox#checkBox_mainScreen::indicator:checked
{
	image: url(:/resource/icon/checked.png);
    width:18px;
    height:18px;
	color: rgb(211, 215, 207);
}</string>
            </property>
            <property name="text">
             <string>  主屏幕</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_main">
            <property name="styleSheet">
             <string notr="true">color: rgb(243, 243, 243);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_rightScreen">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox#checkBox_rightScreen{
	color: rgb(243, 243, 243);
font:18px;
}

QCheckBox#checkBox_rightScreen::indicator {
    /* 选择框尺寸 */
    width:18px;
    height:18px;
}
/*这是对CheckBox的样式进行设定*/
QCheckBox#checkBox_rightScreen::indicator:unchecked
{
    image: url(:/resource/icon/unchecke.png);
    width:18px;
    height:18px;
}
QCheckBox#checkBox_rightScreen::indicator:checked
{
	image: url(:/resource/icon/checked.png);
    width:18px;
    height:18px;
}</string>
            </property>
            <property name="text">
             <string>  右屏幕</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_right">
            <property name="styleSheet">
             <string notr="true">color: rgb(243, 243, 243);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cancel">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ok">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
